cli/commands/build.js

```diff
@@ -20,6 +32,9 @@ const mkdir = dir =>
 
 const compile = async (source, dest) => {
   dest = dest || path.join(source, ".clio", "target", "node");
+  if (!fs.existsSync(dest)) {
+    mkdirp(dest);
```

Comment:
This isn't needed. Each time a file is compiled the directory for it will get recursively created.
See line 42 and 46 https://github.com/clio-lang/clio/blob/91b9751e4ecf1bbac021c323f5b823398f2e5c29/cli/commands/build.js#L42-L46